### PR TITLE
NanoUI tweak

### PR DESCRIPTION
**Pull request by SweetBlueSylveon**
Allows simple mobs to interact with nanoUI if they have "have_hands = TRUE".

### CODE REVIEW

File: vorestation.dme [hunk 1]
```diff
@@ -783,6 +783,7 @@
 #include "code\game\machinery\holoposter.dm"
 #include "code\game\machinery\holosign.dm"
 #include "code\game\machinery\igniter.dm"
+#include "code\game\machinery\isatooluserfix_vr.dm"
```
Comment (Heroman3003) [inline vorestation.dme:786]: Imo better file/path to put the fix in would be "code\modules\nano\interaction\default_vr.dm" for naming/file location consistency.

Comment (SweetBlueSylveon) [reply vorestation.dme:786]: isatooluser as far as I know is defined in machinery, hence why I put it there.

Comment (Heroman3003) [reply vorestation.dme:786]: Yes, but you are adding/changing a default_can_use_topic proc, which is defined in default.dm (in nano files)

File: code/modules/nano/interaction/default_vr.dm [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/mob/living/simple_mob/default_can_use_topic(var/src_object)
+	. = shared_nano_interaction(src_object)
+	if(. != STATUS_CLOSE)
+		. = min(., shared_living_nano_distance(src_object))
+
+//Allows simple mobs to interact with nanoui as long as they have "has_hands = TRUE"
```
Comment (Arokha) [inline code/modules/nano/interaction/default_vr.dm:6]: But you aren't checking has_hands or anything

**Pull request merged**